src/bin/lfortran.cpp

```diff
@@ -1782,6 +1784,7 @@ int main(int argc, char *argv[])
 
         std::string arg_pywrap_file;
         std::string arg_pywrap_array_order="f";
+        std::string extra_runtime_lib_path="";
```

Comment:
```suggestion
        std::vector<std::string> extra_rpaths="";
```
See https://github.com/lfortran/lfortran/issues/1998#issuecomment-1644953716.

Reply:
Actually, let's use `linker_flags` instead of `extra_rpaths`.